src/SkipTracebackWidget.ts

```diff
@@ -19,8 +19,16 @@ interface IError {
   traceback: string[]; // The traceback will contain a list of frames, represented each as a string.
 }
 
-const messageInCollapsedState = (data: IError) => {
-  return `<span class="${RED_BOLD_TEXT_CLASS}">${data.ename}</span>: ${data.evalue}`;
+const addMessageInCollapsedState = (
+  shortError: HTMLPreElement,
+  data: IError
+) => {
+  const eName = document.createElement('span');
```

Comment:
We should ensure if this is ever called twice in a row, it has the came content. Before that was true since it set a value. Now `addMessageInCollapsedState(...); addMessageInCollapsedState(...)` will add data twice. I'm don't see when this would happen, but it's a safer design